docs/docs-content/troubleshooting/enterprise-install.md

```diff
@@ -156,3 +156,44 @@ automatically resolve this issue. If you have self-hosted instances of Palette i
 
    Events:  <none>
    ```
+
+## Scenario - "Too Many Open Files" in Management Cluster
+
+When viewing logs for enterprise management clusters or management clusters using a
+[Private Cloud Gateway](../clusters/pcg/pcg.md), you may encounter a "too many open files" error, which prevents logs
+from tailing after a certain point. To resolve this issue, you must increase the maximum number of file descriptors for
+each node on your cluster.
+
+### Debug Steps
+
+Repeat the following process for each node in your management cluster.
+
+1. Log in to a node in your management cluster.
+
+   ```bash
+   ssh -i <key-name> <user@hostname>
+   ```
+
+2. Switch to `sudo` mode.
+
+   ```bash
+   sudo --login
+   ```
+
+3. Increase the maximum number of file descriptors that the kernel can allocate system-wide.
+
+   ```bash
+   echo "fs.file-max = 1000000" > /etc/sysctl.d/99-maxfiles.conf
+   ```
+
+4. Apply the updated `sysctl` settings.
+
+   ```bash
+   sysctl -p /etc/sysctl.d/99-maxfiles.conf
+   ```
+
+5. Restart the `kubelet` and `containerd` services.
+
+   ```bash
+   systemctl restart kubelet containerd
+   ```
```

Comment:
@achuribooks You may want to include a validation check after this steps to ensure the settings have changed.

For example:

```shell
sysctl fs.file-max
```